GREEDY/Models/GraphData.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GREEDY.Models
+{
+    public class GraphData
+    {
+        public decimal ReceiptPrice { get; set; }
+        public decimal ItemPrice { get; set; }
+        public string Time { get; set; }
+        public DateTime Date { get; set; }
+        public string FullDateTimeString { get { return Date.ToString("yyyy-MM-dd HH:mm"); } }
```

Comment:
we do not take hours. Why you need them?

Reply:
For the graphs. It is for testing purposes only now (If we don't want graphs with hours I will redo later)